src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -45,132 +47,109 @@ export enum SleeveType {
-
-		this.addEventHook('applybuff', {abilityId: this.DRAWN_ARCANA, by: 'player'}, this.onDrawnStatus)
-		this.addEventHook('removebuff', {abilityId: this.DRAWN_ARCANA, by: 'player'}, this.offDrawnStatus)
-		this.addEventHook('death', {to: 'player'}, this.onDeath)
-	}
-
-	override normalise(events: Event[]) {
-		// The whole point of this module was to support "pre-5.3" Three-card Sleeve Draw
-		if (!this.parser.patch.before('5.3')) {
-			return events
-		}
-
-		const startTime = this.parser.fight.start_time
-		let prepullSleeve = true
-		let prepullSleeveFabbed = false
-		const sleeveDrawLog: CastEvent[] = []
-		for (const event of events) {
-			if (event.timestamp < startTime && event.type === 'cast' && this.data.actions.SLEEVE_DRAW.id === event.ability.guid) {
-				// sleeve was already fabbed
-				prepullSleeveFabbed = true
-			}
-			if (event.timestamp - startTime >= this.data.statuses.SLEEVE_DRAW.duration) {
-				// End loop if: 1. Max duration of sleeve draw status passed
-				break
-			} else if (event.type === 'cast'
-				&& this.data.actions.SLEEVE_DRAW.id === event.ability.guid
-				&& event.timestamp >= startTime) {
-				// they used sleeve so it can't have been prepull
-				prepullSleeve = false
-			} else if (event.type === 'cast' && this.data.actions.DRAW.id === event.ability.guid) {
-				sleeveDrawLog.push(event)
-			} else {
-				continue
-			}
-		}
-
-		// if they drew 2-3 cards in the first 30 sec, we can assume they had a sleeve draw
-		if (prepullSleeve && _.inRange(sleeveDrawLog.length, SleeveType.TWO_STACK, SleeveType.TWO_STACK + 2)) {
-			this.cardStateLog[0].sleeveState = sleeveDrawLog.length
-			// prefab a sleeve draw if there isn't already
-			if (!prepullSleeveFabbed) {
-				events.splice(0, 0, {
-					ability: {
-						abilityIcon: sleeveDrawLog.length > 2 ? '019000/019562.png' : '019000/019561.png',
-						guid: 7448,
-						name: 'Sleeve Draw',
-						type: 1,
-					},
-					sourceID: this.parser.player.id,
-					sourceIsFriendly: true,
-					targetID: this.parser.player.id,
-					targetIsFriendly: true,
-					type: 'cast',
-					timestamp: startTime,
-				})
-			}
+		PLAY.forEach(key => { this.play.push(this.data.actions[key].id) })
+		ARCANA_STATUSES.forEach(key => { this.arcanaStatuses.push(this.data.statuses[key].id) })
+		CARD_GRANTING_ABILITIES.forEach(key => { this.cardGrantingAbilities.push(this.data.actions[key].id) })
+		CARD_ACTIONS.forEach(key => { this.cardActions.push(this.data.actions[key].id) })
+		CROWN_PLAYS.forEach(key => { this.crownPlays.push(this.data.actions[key].id) })
+		DRAWN_ARCANA.forEach(key => { this.drawnArcana.push(this.data.statuses[key].id) })
+		CELESTIAL_SEAL_ARCANA.forEach(key => { this.celestialSealArcana.push(this.data.actions[key].id) })
+		LUNAR_SEAL_ARCANA.forEach(key => { this.lunarSealArcana.push(this.data.actions[key].id) })
+		SOLAR_SEAL_ARCANA.forEach(key => { this.solarSealArcana.push(this.data.actions[key].id) })
```

Comment:
Prefer the this.play = PLAY.map(key => this.data.actions[key].id) pattern here for consistency & performance

Reply:
updated based on above